### PR TITLE
fix: `Element.attachShadow()`'s options is missing an attribute.

### PR DESCRIPTION
**Pull request by OnlyWick**
#### Supporting details

- https://dom.spec.whatwg.org/#interface-shadowroot
- https://dom.spec.whatwg.org/#interface-element

```IDL
dictionary ShadowRootInit {
  required [ShadowRootMode](https://dom.spec.whatwg.org/#enumdef-shadowrootmode) mode;
  [boolean](https://webidl.spec.whatwg.org/#idl-boolean) delegatesFocus = false;
  [SlotAssignmentMode](https://dom.spec.whatwg.org/#enumdef-slotassignmentmode) slotAssignment = "named";
};
```

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/element/attachshadow/index.md [hunk 1]
```diff
@@ -79,6 +79,10 @@ attachShadow(options)
       - : A boolean that, when set to `true`, specifies behavior that mitigates custom element issues around focusability.
         When a non-focusable part of the shadow DOM is clicked, the first focusable part is given focus, and the shadow host is given any available `:focus` styling.
 
+    - `slotAssignment`
+      - : default: `named`.
+      - : An enum type, it has two attribute values, `manual` and `named`. When set `slotAssignment` to `manual`, we need to use {{DOMxRef("HTMLSlotElement.assign()")}} to manual assign `slot`.
```
Comment (jpmedley) [inline files/en-us/web/api/element/attachshadow/index.md:84]:
```suggestion
      - : Either `manual` or `named` (default). When set to `manual`, use {{DOMxRef("HTMLSlotElement.assign()")}} to assign a value to`slot`.
```

Comment (jpmedley) [reply files/en-us/web/api/element/attachshadow/index.md:84]: Thank you for the submission. This is more typical of how we do this.

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by jpmedley** [comment]
May I ask why you're not taking the suggestion?

**Comment by OnlyWick** [comment]
> May I ask why you're not taking the suggestion?

sorry, I clicked it wrong on my mobile phone!

**Comment by OnlyWick** [comment]
I have accepted the suggestion.

Joe Medley ***@***.***> 于2022年9月2日周五 00:08写道：

**Review by jpmedley — APPROVED** [review]

**Pull request merged**